src/style.css

```
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 10px;
+  margin-right: 20px;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
+  line-height: 60px;
+}
+
+.nav__item:last-child {
+  margin: 0;
```

```suggestion
  margin-right: 0;
```
Reset only needed margin